### PR TITLE
Add a warning when the user has >= specified amount of warns

### PR DESCRIPTION
**Pull request by DarkView**
Same as #4 but clean (as in no commit mess)

### CODE REVIEW

File: src/plugins/ModActions.ts [hunk 1]
```diff
@@ -444,6 +446,14 @@ export class ModActionsPlugin extends ZeppelinPlugin<IModActionsPluginConfig> {
       ),
     );
 
+    const priorWarnAmount = (await casesPlugin.getCaseTypeAmountForUserId(memberToWarn.id, CaseTypes.Warn)) - 1;
+    if (priorWarnAmount >= config.warn_amount_notify_threshhold) {
+      this.sendErrorMessage(
```
Comment (Dragory) [inline src/plugins/ModActions.ts]: Perhaps we should ask for verification instead? Kinda like we do above for "Failed to message the user".

Comment (DarkView) [reply src/plugins/ModActions.ts]: Sounds like a good idea. Will do on the 13th/14th

**Comment by DarkView** [comment]
Changes made and tested. Sorry for the formatting things, those didnt even show up in my commit preview

**Comment by DarkView** [comment]
Replaced by #36

**Pull request closed without merging**